testdata/includes_yaml/included_with_custom_file.txt

```diff
@@ -0,0 +1 @@
+included_with_custom_file
```

Comment:
These 3 generated `.txt` files should be on a local `.gitignore` file and not commited.